server/service/apple_mdm_test.go

```diff
@@ -403,12 +411,15 @@ func TestAppleMDMAuthorization(t *testing.T) {
 				ctx = test.UserContext(ctx, c.user)
 				_, err = svc.GetMDMAppleCommandResults(ctx, c.cmdUUID)
 				checkAuthErr(t, err, c.shoudFailWithAuth)
+
+				_, err = svc.GetMDMCommandResults(ctx, c.cmdUUID)
```

Comment:
In hindsight, I probably should have extracted this test to the shared `mdm_test` file so that it doesn't get accidentally removed in the future when the deprecated Apple endpoint is finally removed. Would you mind either moving it or adding a TODO in the `mdm_test` file?